scripts/watch.js

```diff
@@ -0,0 +1,74 @@
+const fs = require( "fs" );
+const http = require( "http" );
+const path = require( "path" );
+
+const rollup = require( "rollup" );
+const loadConfigFile = require( "rollup/dist/loadConfigFile" );
+
+const server = http.createServer( ( req, res ) => {
+	try {
+		const requestedPath = path.format( {
+			root: process.cwd(),
+
+			// path.normalize will remove any "..".
+			// Probably useless as at least Chrome resolves any ".."
+			// _before_ sending the request.
+			// Probably still a good idea to avoid accessing anything
+			// _outside_ of process.cwd() anyways.
+			base: path.normalize( req.url )
```

Comment:
At first glance, this did not appear to have the intended effect:

```
> path.normalize('../bar/quuz');
'../bar/quuz'

> path.format({ root: '/tmp/', base: '../bar/quuz' });
'/tmp/../bar/quuz'
```

But I understand now, that it works so long as the normalize input has a leading slash:
```
> path.normalize('/../../bar/quuz');
'/bar/quuz'
```

Reply:
As such, it seems `path.format()` and its `root` option appear not add benefits or restrictions compared to the simpler `path.join(cwd, normalized)`. Is that right, or is there a different reason for using `path.format()`?

Reply:
Updated to use `path.join`, indeed there was no use for `path.format`, my bad.